packages/dashboards/tests/acceptance/dashboards-test.js

```diff
@@ -662,4 +662,140 @@ module('Acceptance | Dashboards', function(hooks) {
       'the filters reflect the dashboards modified filters'
     );
   });
+
+  test('New widget after clone', async function(assert) {
```

Comment:
This test feels pretty big, maybe cut it down to just clone a dashboard, add a new widget and assert it was added instead of all the in between asserts

There's a lot of overlap with these two tests

`clone dashboard`
https://github.com/yahoo/navi/blob/7f8ebd603a6e56f1338c38039563e9e9c8bb9962/packages/dashboards/tests/acceptance/dashboards-test.js#L327-L354

`New widget`
https://github.com/yahoo/navi/blob/7f8ebd603a6e56f1338c38039563e9e9c8bb9962/packages/dashboards/tests/acceptance/dashboards-test.js#L368-L478